src/Diagnostics.RuntimeHost/Controllers/DiagnosticControllerBase.cs

```diff
@@ -304,6 +304,7 @@ protected async Task<HostingEnvironment> GetHostingEnvironment(string subscripti
 
             HostingEnvironment hostingEnv = new HostingEnvironment(subscriptionId, resourceGroup, name)
             {
+                Name = stampPostBody.InternalName,
```

Comment:
FriendlyName?

Reply:
So in the world of ASE there are two names the internal name and friendly name. The friendly name is an easier to remember ASE name which is the name that customer set and the ugly name aka internal name is the waws-prod- prefixed name.

Reply:
Thanks Hawk for the explanation:)